kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,117 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+export const title = 'Payment terms';
```

Comment:
It's still not consistent. Here it's `Payment terms` and in the second paragraph it's `These Payment Terms cover your...`

Reply:
Ugh sorry some of my changes to `terms.jsx` got lost in the rebase.